### DISCUSSION TITLE
Add warning of purchasing device from Verizon (and potentially other carriers)

### DISCUSSION THREAD

**Discussion by StormLight14** [zenfyrdev/bootloader-unlock-wall-of-shame#40] [Ideas]
After purchasing a Pixel 6 Pro from Verizon because of a crazy cheap deal, I found out that they completely lock down the bootloader and there is virtually nothing you can do. I'm not sure if this is unique to them, but it made me pretty pissed.

**Comment by zenfyrdev** [comment]
This is a fairly common practice. I'll mention it in the README, thanks for reminding me.
